backends/ebpf/tests/ptf/test.py

```diff
@@ -595,404 +592,112 @@ def runTest(self):
-        exp_pkt = testutils.simple_ip_packet(eth_dst="11:22:33:44:55:66")
-        exp_pkt[Ether].type = 0x8601
 
-        testutils.send_packet(self, PORT0, pkt)
-        testutils.verify_packet(self, exp_pkt, PORT1)
+        # flow rules 'tbl_ternary_3':
+        # 1. ipv4.protocol=0x7, ipv4.diffserv=selector, ipv4.dstAddr=0xffffffff^0xffffffff => action 0 priority 1
+        # 2. ipv4.protocol=0x7, ipv4.diffserv=selector, ipv4.dstAddr=0x0^0x0 => action 1 priority 10
+        ref1 = self.action_selector_add_action(selector="ingress_as", action=0, data=[])
+        ref2 = self.action_selector_add_action(selector="ingress_as", action=1, data=[])
+        self.table_add(
+            table="ingress_tbl_ternary_3",
+            key=["0xffffffff^0xffffffff", "0x7"],
+            references=[ref1],
+            priority=1,
+        )
+        self.table_add(
+            table="ingress_tbl_ternary_3",
+            key=["0x0^0x0", "0x7"],
+            references=[ref2],
+            priority=10,
+        )
 
-        # Validate that cache entry is used during packet processing. By altering cache we get two different states
-        # of a table entry with different executed actions. Based on this it is possible to detect which entry is in
-        # use, table entry or cached entry. If we only read cache here then we couldn't test that it is used correctly.
-        self.table_update(
-            table="ingress_tbl_ternary_cache",
-            key=["00:11:22:33:44:55"],
-            action=1,
-            data=["32w0", "11:22:33:44:55:66", "16w0", "64w0"],
+        # flow rules 'tbl_ternary_4':
+        # 2. hdr.ethernet.srcAddr=00:00:33:44:55:00^00:00:FF:FF:FF:00 => action 1 priority 10
+        ref3 = self.action_profile_add_action(ap="ingress_ap", action=1, data=[])
+        self.table_add(
+            table="ingress_tbl_ternary_4",
+            key=["00:00:33:44:55:00^00:00:FF:FF:FF:00"],
+            references=[ref3],
+            priority=10,
         )
-        testutils.send_packet(self, PORT0, pkt)
-        exp_pkt[Ether].type = 0x0800
-        testutils.verify_packet(self, exp_pkt, PORT1)
 
-        # Delete table entry to test if NIKSS library invalidates cache
-        self.table_delete(table="ingress_tbl_ternary", key=["00:11:22:33:44:55"])
+        pkt = testutils.simple_udp_packet(
+            eth_src="11:22:33:44:55:66", ip_src="1.2.3.4", ip_dst="192.168.2.1"
+        )
         testutils.send_packet(self, PORT0, pkt)
+        pkt[Ether].type = 0x1122
+        pkt[IP].proto = 0x7
+        pkt[IP].tos = 0x5
+        pkt[IP].chksum = 0xB3E7
+        pkt[IP].src = "17.17.17.17"
+        pkt[IP].dst = "255.255.255.255"
+        pkt[UDP].chksum = 0x044D
         testutils.verify_packet(self, pkt, PORT1)
 
 
-class WideFieldTableSupport(P4EbpfTest):
-    """
-    Test support for fields wider than 64 bits in tables using IPv6 protocol.
-    """
-
-    p4_file_path = "p4testdata/wide-field-tables.p4"
+@unittest.skipIf(
```

Comment:
Why are you adding a decorator to skip test `ActionDefaultTernaryPSATest`?  It was passing before, I thought, so I cannot think of any reason to add this.